### ISSUE TITLE
Allow the user to specify which provider to open on the command line

### ISSUE THREAD

**Issue by bdutro** [gort818/qtwebflix#30]
It would be nice if I could specify which provider I want to open on the command line. Something like:
```
qtwebflix --provider=netflix
```
I'm looking at using this with Kodi for Netflix/Amazon/etc. playback and this would allow me to make separate shortcuts for each provider.

**Comment by bdutro** [comment]
I'm aware of the context menu, it would just be nice to skip the extra step of navigating to a different provider if I know I want to go to a particular one (especially since right now it will open up the last page I visited). Right now, my flow would have to go something like this:

1. I watch Netflix, then close qtwebflix when I'm done.
2. I want to watch Amazon Prime Video, so I launch qtwebflix.
3. qtwebflix goes back to Netflix.
4. I have to manually navigate to Amazon Prime Video with the context menu.

It would be easier if I could instead:

1. I watch Netflix with qtwebflix --provider=netflix and close it when I'm done.
2. I want to watch Amazon Prime Video, so I run qtwebflix --provider=amazon and it goes directly to Amazon.

> How would you use this with Kodi? never thougth about it

Kodi has a Docker addon, so I'm putting together a Docker image with qtwebflix. Then I could use something like the Advanced Emulator Launcher addon to create shortcuts to my preferred streaming sites. I'm using a third-party LibreELEC build that includes Chrome for all of this at the moment, but it would be nice to be able to use an official build with something a little more light weight.

**Comment by gort818** [comment]
Ok I see your point I might work on something this week, we can have command line options and context menu

**Comment by gort818** [comment]
@bdutro 
I just updated a test branch adding command line options
```
Options:
  -h, --help                   Displays this help.
  -v, --version                Displays version information.
  -p, --provider <provider>    Set content provider eg. netflix.com
  -u, --useragent <useragent>  change useragent eg. "Mozilla/5.0 (Windows NT
                               10.0; WOW64) AppleWebKit/537.36 (KHTML, like
                               Gecko) Chrome/67.0.2526.73 Safari/537.36"
```
https://github.com/gort818/qtwebflix/tree/commandline

**Comment by gort818** [comment]
This has been merged into master give it a try!

**Comment by gort818** [comment]
closed with 735fbd278230dad0f1ade9cfae96a9f8a4c83aab
